lib/archethic/metrics/parser.ex

```diff
@@ -5,6 +5,79 @@ defmodule ArchEthic.Metrics.Parser do
+          ...>     archethic_db_duration_count{query='get_transaction'} 2061
+          ...>     # HELP archethic_contract_parsing_duration
+          ...>     # TYPE archethic_contract_parsing_duration histogram
+          ...>     archethic_contract_parsing_duration_bucket{le='0.001'} 153
+          ...>     archethic_contract_parsing_duration_bucket{le='+Inf'} 177
+          ...>     archethic_contract_parsing_duration_sum 0.12
+          ...>     archethic_contract_parsing_duration_count 177
+          ...>     # HELP archethic_mining_full_transaction_validation_duration
+          ...>     # TYPE archethic_mining_full_transaction_validation_duration histogram
+          ...>     archethic_mining_full_transaction_validation_duration_bucket{le='0.01'} 0
+          ...>     archethic_mining_full_transaction_validation_duration_bucket{le='+Inf'} 85
+          ...>     archethic_mining_full_transaction_validation_duration_sum 31.9
+          ...>     archethic_mining_full_transaction_validation_duration_count 85
+          ...>     # HELP archethic_mining_fetch_context_duration
+          ...>     # TYPE archethic_mining_fetch_context_duration histogram
+          ...>     archethic_mining_fetch_context_duration_sum 8.2
+          ...>     archethic_mining_fetch_context_duration_count 88
+          ...>     # HELP archethic_mining_pending_transaction_validation_duration
+          ...>     # TYPE archethic_mining_pending_transaction_validation_duration histogram
+          ...>     archethic_mining_pending_transaction_validation_duration_bucket{le='0.001'} 20
+          ...>     archethic_mining_pending_transaction_validation_duration_bucket{le='+Inf'} 87
+          ...>     archethic_mining_pending_transaction_validation_duration_sum 11.4
+          ...>     archethic_mining_pending_transaction_validation_duration_count 87
+          ...>     # HELP archethic_election_storage_nodes_duration
+          ...>     # TYPE archethic_election_storage_nodes_duration histogram
+          ...>     archethic_election_storage_nodes_duration_bucket{nb_nodes='1',le='+Inf'} 1074
+          ...>     archethic_election_storage_nodes_duration_sum{nb_nodes='1'} 0.032
+          ...>     archethic_election_storage_nodes_duration_count{nb_nodes='1'} 1074
+          ...>     archethic_election_storage_nodes_duration_bucket{nb_nodes='2',le='0.001'} 120
+          ...>     archethic_election_storage_nodes_duration_sum{nb_nodes='2'} 0.011
+          ...>     archethic_election_storage_nodes_duration_count{nb_nodes='2'} 122
+          ...>     archethic_election_storage_nodes_duration_sum{nb_nodes='3'} 3.11
+          ...>     archethic_election_storage_nodes_duration_count{nb_nodes='3'} 45719
+          ...>     # HELP vm_memory_atom
+          ...>     # TYPE vm_memory_atom gauge
+          ...>     vm_memory_atom 1589609
+          ...>     """
+          iex> length(ArchEthic.Metrics.Parser.run(data))==Enum.count(ArchEthic.Metrics.Parser.run(data),fn x ->
```

Comment:
I think you can simplify, but just expecting the real result, without the Enum.count fn, etc...

Reply:
Also If you want you can provide different test cases, maybe one with histogram, one with gauge,  one with labels, ..., 
this will reduce the size of the test